web/src/shared/helpers/utils.ts

```diff
@@ -130,3 +131,19 @@ export const applyTableConfigs = <TFilter, TSorting, TDirection, TOperator, TVal
 };
 export const mapUndefString = (fn: (s: string) => void) => (val: string | undefined) =>
     fn(val || '');
+
+export const getTaxonsId = (searchResult: any[] = []) => {
```

Comment:
Could you please try to avoid using any and specify type if possible?

Reply:
done: function, no problem!